doc/manual/cookbooks/global_melt/global_melt.prm

```diff
@@ -21,14 +21,14 @@ set Nonlinear solver tolerance             = 1e-5
 # time scale than the flow of melt, so we want to decouple the advection
 # of melt (and temperature) and the melting process itself. To do that, 
 # we use the operator splitting scheme, and define that for every 
-# advection time step, we want to do at least 50 reaction time step, 
-# and if these time steps would be larger than 100,000 years, we will do 
+# advection time step, we want to do at least 10 reaction time steps, 
+# and if these time steps would be larger than 10,000 years, we will do 
```

Comment:
If you like the suggest rewording in the parameter file, consider rewording here as well. Completely up to you.